vaadin-legacy-components-flow/pom.xml

```
@@ -34,6 +34,10 @@
             <artifactId>flow-data</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>com.vaadin</groupId>
+            <artifactId>flow-html-components</artifactId>
```

At the moment only needed for tests ? 
```suggestion
            <artifactId>flow-html-components</artifactId>
            <scope>test</scope>
```